src/test/java/org/amazon/base/BaseMethods.java

```diff
@@ -0,0 +1,45 @@
+package org.amazon.base;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+import org.amazon.library.PropertyReader;
+import org.amazon.pages.HomePage;
+import org.amazon.pages.LoginPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeTest;
+
+public class BaseMethods {
+	
+	public WebDriver driver;
```

Comment:
Instance variables should never be marked as public in a class. If they are needed to be used in other classes, expose them through appropriate getter methods.

Reply:
Done.